python/django-tutorial-docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+services:
+  app:
+    depends_on:
+      - agent
+      - db
+    build: .
+    command: ddtrace-run python manage.py runserver 0.0.0.0:8000
+    environment:
+      - 'DATADOG_TRACE_AGENT_HOSTNAME=agent'
+      - 'DD_LOGGING_RATE_LIMIT=0'
+    ports:
+      - '8000:8000'
+
+  db:
+    image: postgres
+
+  agent:
+    image: 'datadog/agent'
+    environment:
+      - 'DD_HOSTNAME=trace-example'
+      - 'DD_TAGS=env:dev'
+      - 'DD_API_KEY=${DD_API_KEY}'
+      - 'DD_APM_ENABLED=true'
+      - 'DD_LOGS_ENABLED=true'
+      - 'DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL=true'
+      - 'DD_AC_EXCLUDE="name:datadog-agent"'
```

Comment:
if you want to exclude this container it'll be like `django-tutorial-docker_agent` or something like that.

Probably can just remove this and the logs stuff too, unless you are collecting/shipping logs up.